### PR TITLE
adding Forty Day8 file

### PR DESCRIPTION
**Pull request by Poornima545**

### CODE REVIEW

File: FortyDay8/index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <title>Forty HTML5Up</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section id="one">
+    <div class="main">
+        <header id="header">
+            <a href="#" class="logo">
+                <strong>forty</strong>
```
Comment (HEMASINCHU) [inline FortyDay8/index.html:15]: use font weight property

File: FortyDay8/index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                <span>by html5 up<a href="#menu">menu</a></span>
+            </a>    
+        </header>
+    </div>
+        <header class="major">
+            <h1>Hi, my name is Forty</h1>
+            <hr style="width: 70%; margin-left: 0; height: 0.10em"/>
```
Comment (HEMASINCHU) [inline FortyDay8/index.html:22]: instead of hr use border bottom property

File: FortyDay8/index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            <p>Nullam et orci eu lorem consequat tincidunt vivamus et sagittis libero. Mauris aliquet magna magna sed nunc rhoncus pharetra. Pellentesque condimentum sem. In efficitur ligula tate urna. Maecenas laoreet massa vel lacinia pellentesque lorem ipsum dolor. Nullam et orci eu lorem consequat tincidunt. Vivamus et sagittis libero. Mauris aliquet magna magna sed nunc rhoncus amet pharetra et feugiat tempus.</p>
+            <a href="#" class="getstarted">Get started </a>
+        </div>
+    </section> 
+    <section id="three">
+        <form action="#">
+            <label for="name">name</label>
+            <input type="text" name="name" id="name" required>
+            <label for="email">email</label>
+            <input type="text" id="email" id="email" required><br>
```
Comment (HEMASINCHU) [inline FortyDay8/index.html:44]: check 44 line type is email , delete one id  and don't use br  tag

File: FortyDay8/index.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            <label for="message">message</label><br>
+            <textarea name="message" id="" cols="60" rows="10">
+            </textarea><br>
+            <input type="submit" value="send message" id="send message">
+            <input type="reset" value="Clear">
+        </form>
+        </section>
+        <section id="four">
+        <div class="contact">
```
Comment (HEMASINCHU) [inline FortyDay8/index.html:53]: use padding

**Review by HEMASINCHU — COMMENTED** [review]
fix the comments

**Comment by HEMASINCHU** [comment]
fix the above comments

**Comment by Poornima545** [comment]
wil open again

**Pull request closed without merging**